packages/vgplot/src/directives/interactors.js

```diff
@@ -20,16 +20,16 @@ export function toggle({ as, ...rest }) {
   return interactor(Toggle, { ...rest, selection: as });
 }
 
-export function toggleX({ as }) {
-  return toggle({ as, channels: ['x'] });
+export function toggleX({ as, ...rest }) {
+  return toggle({ as, ...rest, channels: ['x'] });
```

Comment:
We can simplify the signature and implementation to:

```js
export function toggleX(options) {
  return toggle({ ...options, channels: ['x'] });
}
```

...and similarly for the other modified `toggle` methods.